src/main/java/com/naitei/group3/movie_ticket_booking_system/controller/admin/MovieController.java

```diff
@@ -92,6 +92,10 @@ public String listMovies(
     @GetMapping("/{id}")
     public String getMovieDetail(@PathVariable Long id, Model model) {
         MovieDTO movie = movieService.getMovieById(id);
+        if (movie == null) {
+            return "errors/404";
+        }
```

Comment:
```suggestion
        if (movie == null) {
            return "errors/404";
        }
```

thực ra ko sai, nhưng chị thích sử dụng theo dạng Global Exception hơn
vì nó xử lý tập trung


ở Global Exception thì check request là api hay web mvc thì mình trả ra cho hặp lý thôi